### DISCUSSION TITLE
Is there a badge I can add to a repo indicating the results of golangci-lint?

### DISCUSSION THREAD

**Discussion by MichaelSnowden** [golangci/golangci-lint#3431] [Q&A] [answered]
I'd like to add a badge to a repo that contains something like a score, based on the results of running this linter. Does something like that exist? I want something on my repo that indicates the quality of the code as determined by this linter, and I'd rather not have to set up another tool like code climate to do this

**Comment by ldez** [comment, marked as answer]
No, because golangci-lint doesn't provide a score: golangci-lint is a meta linter, and the linting rules depend on linters that the user chooses to use.
All the issues reported by golangci-lint must be fixed, your CI status is the only possible indicator.

So it's impossible to provide a score.

**Comment by Antonboom** [comment]
@MichaelSnowden maybe https://github.com/gojp/goreportcard will be useful for you.
